carp.studies.core/src/commonMain/kotlin/dk/cachet/carp/studies/application/ParticipantService.kt

```diff
@@ -0,0 +1,60 @@
+package dk.cachet.carp.studies.application
+
+import dk.cachet.carp.common.EmailAddress
+import dk.cachet.carp.common.UUID
+import dk.cachet.carp.studies.domain.ParticipantGroupStatus
+import dk.cachet.carp.studies.domain.users.AssignParticipantDevices
+import dk.cachet.carp.studies.domain.users.Participant
+
+
+/**
+ * Application service which allows adding participants to studies and creating deployments for them.
+ */
+interface ParticipantService
```

Comment:
Nothing really new here. These are just methods moved from `StudyService` to this new interface.